wwwapp/views.py

```diff
@@ -528,17 +542,17 @@ def article_name_list_view(request):
 @login_required()
 def your_workshops_view(request):
     workshops = Workshop.objects.filter(lecturer__user=request.user)
-    return render_workshops(request, 'Twoje warsztaty', workshops)
+    return render_workshops(request, 'Twoje warsztaty', True, workshops)
 
 
 @login_required()
 @permission_required('wwwapp.see_all_workshops', raise_exception=True)
 def all_workshops_view(request):
     workshops = Workshop.objects.all()
-    return render_workshops(request, 'Wszystkie warsztaty', workshops)
+    return render_workshops(request, 'Wszystkie warsztaty', False, workshops)
 
 
-def render_workshops(request, title, workshops):
+def render_workshops(request, title, link_to_edit, workshops):
```

Comment:
Why do we pass booleans to the link_to_edit parameter. Add typespecs.

Reply:
Becouse it's a boolean.